docs/tutorials/timeseries/forecasting-model-zoo.md

```diff
@@ -48,7 +48,6 @@ Note that some of the models' hyperparameters have names and default values that
    SeasonalNaiveModel
    AverageModel
    SeasonalAverageModel
-   ARIMAModel
```

Comment:
Do we propose to use AutoARIMA as a replacement for the ARIMA model?

Reply:
As of v0.8.2, ARIMA is not used in any of the presets, but I agree that it would be nice to have the plain ARIMA model at some point. The plan would be to return ARIMA after #3458 is resolved and we can upgrade to StatsForecast >=1.5.0 where the plain ARIMA model is introduced. Currently this upgrade is blocked by https://github.com/omry/omegaconf/pull/1114.